internal/reconciler/reconciler.go

```diff
@@ -174,12 +174,6 @@ func (r *Reconciler) doWaitForTermination(_ context.Context, _ logr.Logger, cs *
 }
 
 func (r *Reconciler) ApplyAllSets(ctx context.Context, log logr.Logger, withChangeSet withChangeSetFunc) error {
-	if !r.instanceExists {
```

Comment:
There is exactly the same invocation in `ApplyInstance`, which is actually visible in this diff right above.